inference-engine/src/plugin_api/cpp_interfaces/impl/ie_plugin_internal.hpp

```diff
@@ -56,16 +56,14 @@ class InferencePluginInternal : public IInferencePlugin {
     ~InferencePluginInternal() override = default;
 
 public:
-    void LoadNetwork(IExecutableNetwork::Ptr& executableNetwork, const ICNNNetwork& network,
-                     const std::map<std::string, std::string>& config) override {
-        LoadNetworkImplPrivate(executableNetwork, network, config);
+    ExecutableNetwork LoadNetwork(const ICNNNetwork& network,
```

Comment:
Should we add documentation for these methods?

Reply:
since it's an overridden method, documentation is inherited as well. See interfaces, they have docs.